src/site/content/en/blog/webtransport/index.md

```diff
@@ -130,22 +130,20 @@ await transport.ready;
 
 Once you have a WebTransport instance that's connected to a server, you can use it to send and receive discrete bits of data, known as [datagrams](https://en.wikipedia.org/wiki/Datagram).
 
-The `sendDatagrams()` method returns a <code>[WritableStream](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream)</code>, which a web client can use to send data to the server. The <code>receiveDatagrams()</code> method returns a <code>[ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream)</code>, allowing you to listen for data from the server. Both streams are inherently unreliable, so it is possible that the data you write will not be received by the server, and vice versa.
+The `writeable` getter returns a <code>[WritableStream](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream)</code>, which a web client can use to send data to the server. The <code>readable</code> getter returns a <code>[ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream)</code>, allowing you to listen for data from the server. Both streams are inherently unreliable, so it is possible that the data you write will not be received by the server, and vice versa.
 
 Both types of streams use <code>[Uint8Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array)</code> instances for data transfer.
 
 ```js
 // Send two datagrams to the server.
-const ws = transport.sendDatagrams();
-const writer = ws.getWriter();
+const writer = transport.writeable.getWriter();
```

Comment:
Still wrong.
In Chrome 91 Beta it's `transport.datagramsWritable.getWriter()`.
In W3C spec it's `transport.datagrams.writable.getWriter()`.